include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -857,6 +857,10 @@ struct OrtApi {
      * \param index index of string tensor element to fill 
      */
   ORT_API2_STATUS(FillStringTensorElement, _Inout_ OrtValue* value, _In_ const char* s, size_t index);
+  
+  // Control pre-packing of initialized constant tensors
+  ORT_API2_STATUS(EnablePrePacking, _Inout_ OrtSessionOptions* options);
```

Comment:
You might be able to achieve this without adding these 2 APIs once [this PR](https://github.com/microsoft/onnxruntime/pull/4718) is merged.

Reply:
You may merge first, and I can move the prepacking into session config map in my change

Reply:
You may merge first, and I can move the prepacking into session config map in my change



---
In reply to: [466743313](https://github.com/microsoft/onnxruntime/pull/4413#discussion_r466743313) [](ancestors = 466743313)